### PR TITLE
Add some functions to bitvector

### PR DESCRIPTION
**Pull request by plambein**
Added an AND and OR function. Also the ability to create a 1-only bitvector and a 0-only bitvector of given size. These are handy to work with the AND and OR functions.

### CODE REVIEW

**Comment by pattyshack** [comment]
could you add tests please?  Thanks!

File: container/bitvector/bitvector.go [hunk 1]
```diff
@@ -31,6 +31,50 @@ func (vector *BitVector) Length() int {
 	return vector.length
 }
 
+// And will AND two BitVector's and returns the resulting one
+func (a *BitVector) And(b *BitVector) *BitVector {
+	vector := NewBitVector(make([]byte, 0), 0)
+	if a.length != b.length {
+		return vector
```
Comment (pattyshack) [inline container/bitvector/bitvector.go]: Returning an empty bit vector when the lengths are different seems unhelpful.  When the lengths are different, maybe use to longest vector's length instead (and 0 pad the shorter vector).

Same for Or

**Comment by lexfrei** [comment]
@plambein, do you need any help?

**Comment by orborde** [comment]
OK, looks like the original committer ran away, and there are things we don't like about the patch. Killing it.

@plambein Happy to revisit this PR if you for some reason come back and fix it up! :smile:

**Pull request closed without merging**